dbms/tests/queries/0_stateless/01018_ddl_dictionaries_create.sql

```diff
@@ -0,0 +1,165 @@
+SET send_logs_level = 'none';
+
+DROP DATABASE IF EXISTS database_for_dict;
+
+CREATE DATABASE database_for_dict Engine = Ordinary;
+
+DROP TABLE IF EXISTS database_for_dict.table_for_dict;
+
+CREATE TABLE database_for_dict.table_for_dict
+(
+  key_column UInt64,
+  second_column UInt8,
+  third_column String
+)
+ENGINE = MergeTree()
+ORDER BY key_column;
+
+INSERT INTO database_for_dict.table_for_dict VALUES (1, 100, 'Hello world');
+
+DROP DATABASE IF EXISTS ordinary_db;
+
+CREATE DATABASE ordinary_db ENGINE = Ordinary;
+
+SELECT '=DICTIONARY in Ordinary DB';
+
+DROP DICTIONARY IF EXISTS ordinary_db.dict1;
+
+CREATE DICTIONARY ordinary_db.dict1
+(
+  key_column UInt64 DEFAULT 0,
+  second_column UInt8 DEFAULT 1,
+  third_column String DEFAULT 'qqq'
+)
+PRIMARY KEY key_column
+SOURCE(CLICKHOUSE(HOST 'localhost' PORT 9000 USER 'default' TABLE 'table_for_dict' PASSWORD '' DB 'database_for_dict'))
+LIFETIME(MIN 1 MAX 10)
+LAYOUT(FLAT());
+
+SHOW CREATE DICTIONARY ordinary_db.dict1;
+
+SHOW DICTIONARIES FROM ordinary_db LIKE 'dict1';
+
+EXISTS DICTIONARY ordinary_db.dict1;
```

Comment:
and what will happen if same dict is described both in xml and ddl?

Reply:
I hope we will not be able to load dictionary and got an exception on creation, but I need to add test for this!